dist/s_evergreen

```diff
@@ -16,6 +16,31 @@ test -s $t && {
     cat $t
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
 
+    echo "Check for Python dependencies without specified versions:"
+
+    unspecified_dependency=false
+
+    pip_installs=$(grep -E 'pip(3)? install' "test/evergreen.yml")
+
+    while IFS= read -r line; do
+        for word in $line; do
+            if [[ "$word" =~ ^pip(3)?$ || "$word" == "install" || "$word" =~ ^python(3)?$ || "$word" =~ -.* ]]; then
+                continue
+            fi
+            if [[ ! "$word" =~ .*==.* ]]; then
```

Comment:
On first glance this regex is a bit confusing. Can we have a comment explaining that `==` implies the presence of a version string?
`.*==[0-9.]+` might also be clearer